UnitySDK/UnitySDK.sln.DotSettings

```diff
@@ -0,0 +1,5 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
What is this file for ?

Reply:
Rider project file

Reply:
Yes, this Rider file includes words added to the Rider dictionary.